src-web/nls/platform.properties

```diff
@@ -144,11 +145,28 @@ modal.enforce-hcmcompliance.heading = Enforce policy
 modal.enforce-hcmcompliance.label = Enforce policy
 modal.inform-hcmcompliance.heading = Inform policy
 modal.inform-hcmcompliance.label = Inform policy
-modal.button.submit = Submit
+modal.automation-hcmcompliance.heading = Edit policy violation automatio
 modal.enable.description = Are you sure you want to enable this policy?
 modal.disable.description = Are you sure you want to disable this policy?
 modal.inform.description = Are you sure you want to change the remediation action of this policy to inform? You must manually remediate the policy violations.
 modal.enforce.description = Are you sure you want to enforce this policy? The policy makes any necessary changes to the applied cluster to remediate the violations.
+modal.ansible.automation.description = Modify an existing provider connection, or create a new one to automate your selected policy and violation. You can also schedule the automation to run once, continuously, or disable for now.
+modal.ansible.launch.connection = Launch to connection URL
+modal.ansible.credential.selection.title = Provider connection
+modal.ansible.credential.selection.placeholder = Select Ansible Credential
+modal.ansible.no.credential = Ansible Credential is required to create Ansible Job, create your console here:
+modal.ansible.launch.createCredential = Create a new provider connection
```

Comment:
`provider connection` has been renamed to `credentials` please change it accordingly.